pvlib/irradiance.py

```diff
@@ -551,13 +551,16 @@ def poa_components(aoi, dni, poa_sky_diffuse, poa_ground_diffuse):
 
 
 def get_ground_diffuse(surface_tilt, ghi, albedo=.25, surface_type=None):
-    '''
-    Estimate diffuse irradiance from ground reflections given
-    irradiance, albedo, and surface tilt.
+    r'''
+    Estimate diffuse irradiance on a tilted surface from ground reflections.
+
+    Ground diffuse irradiance is calculated as
+
+    .. math::
+
+       G_{ground} = GHI \times \rho \times \frac{1 - \cos\beta}{2}
 
-    Function to determine the portion of irradiance on a tilted surface
-    due to ground reflections. Any of the inputs may be DataFrames or
-    scalars.
+    where :math:`\rho` is albedo and :math:`\beta` is ``surface_tilt``.
```

Comment:
```suggestion
    where :math:`\rho` is ``albedo`` and :math:`\beta` is ``surface_tilt``.
```

just for consistency